### PR TITLE
Upgrade to SDK 1.7

### PR DESCRIPTION
**Pull request by cocreature**
Nothing super interesting here, mainly got rid of the version header
to fix a warning and while bumping npm deps, I fixed a type error due
to a new version of @types/history.

### CODE REVIEW

**Comment by lucianojoublanc-da** [comment]
This is @GeorgSchneider 's domain, so need the ok from him, but looking at this the `yarn.lock` file has been changed and all the dependencies have been updated. Was that intentional? Or is it automatically pulled in by the 1.7.0 upgrade?

**Comment by cocreature** [comment]
> This is @GeorgSchneider 's domain, so need the ok from him, but looking at this the `yarn.lock` file has been changed and all the dependencies have been updated. Was that intentional? Or is it automatically pulled in by the 1.7.0 upgrade?

Yes, that was intentional. We already had confused users about things like `@types/history` which had the breaking change that I also fixed here. imho deps should be updated frequently. If you want to constraint upgrades add version constraints to your `package.json`.

**Review by lucianojoublanc-da — APPROVED** [review]

**Comment by cocreature** [comment]
thanks for the review! I’ll wait for @GeorgSchneider to also take a look before merging.

**Review by GeorgSchneider — APPROVED** [review]

**Pull request merged**
